quadrat/inc/render-navigation-block.php

```diff
@@ -0,0 +1,35 @@
+<?php
+if ( ! function_exists( 'render_navigation_block' ) ) :
+
+	function render_navigation_block( $menu_location ) {
+		if ( ! has_nav_menu( $menu_location ) ) {
+			return;
+		}
+
+		$has_primary_nav_items = wp_nav_menu(
+			array(
+				'theme_location' => $menu_location,
+				'fallback_cb'    => false,
+				'echo'           => false,
+			)
+		);
+
+		if ( ! $has_primary_nav_items ) {
+			return;
+		}
+
+		$menu_markup  = '<!-- wp:html -->';
+		$menu_markup .= wp_nav_menu(
```

Comment:
I wonder if we can save this in a variable for use in `$has_primary_nav_items` and then just output it here, to save calling `wp_nav_menu` twice.

Reply:
Oh man good call.  Deleted much codes with that change. :)